### PR TITLE
Adds "Backup Security Uniforms" station trait, Black mesa themed security (Bluesec)

### PR DESCRIPTION
**Pull request by Singul0**
## About The Pull Request
Adds "Backup Security Uniforms", a neutral station trait that changes the outfit of all the sec jobs (except det) to be more BMRF security guard inspired (mostly just making them bluer). this is mostly by using each job respective alternate uniforms, only adding the bluesec plasmaman enviro set as the sole clothing addition.
image
also decals the brig blue to fit the theme of the outfits!

## Why It's Good For The Game
I found out about the barney security guard alternate outfit and figured that a station trait that makes security half-life 1 themed instead of star-trek redshirt themed would be cool and fun, considering the sprites were all there already.

## Changelog

:cl:
add: Adds a new station trait "Backup Security Uniforms"
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
Should remove the normal uniforms from the vendors too

**Comment by Singul0** [comment]
> Should remove the normal uniforms from the vendors too

we should probably still give them the choice at the cost of being a fashion pariah, station_trait lore be damned

**Comment by GPeckman** [comment]
These outfits have similar armor values to the default outfits, right?

**Comment by Singul0** [comment]
> These outfits have similar armor values to the default outfits, right?

given all of these outfits are subtypes of normal sec gear, yes

**Comment by norsvenska** [comment]
yes, they have the same armour as the standard uniforms

**Comment by Exester509** [comment]
Can you make the blue plasma gear purchaseable from the vendor? Poor purple skeletons can't go blue without the trait

**Comment by Sadboysuss** [comment]
the envirosuit should only be orderable from cargo as it is with all envirosuits

**Comment by Singul0** [comment]
> the envirosuit should only be orderable from cargo as it is with all envirosuits

doesn't seem right adding a crate solely just for the envirosuits. going to wait on maintainer directions for this

**Comment by Sadboysuss** [comment]
> > the envirosuit should only be orderable from cargo as it is with all envirosuits
> 
> doesn't seem right adding a crate solely just for the envirosuits. going to wait on maintainer directions for this

there is no other vendor that sells envirosuits so this isn't just a cosmetic change, but a balance change too because a plasmaman sec off can get a replacement from the secvendor in case their uniform is destroyed, then it become a consistency issue of: "why do we only have sold envirosuits for sec off? why not other jobs?"

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -584,3 +584,34 @@
 	dynamic_category = RULESET_CATEGORY_NO_WITTING_CREW_ANTAGONISTS
 	threat_reduction = 15
 	dynamic_threat_id = "Background Checks"
+
+/// Black mesa security
+/datum/station_trait/blue_sec
+	name = "Backup Security Uniforms"
+	report_message =  "Due to problems in our logistical network of your sector, we have been forced to equip your security department with uniforms from our subsidiary scientific research company. Brig has been recolored accordingly"
+	trait_type = STATION_TRAIT_NEUTRAL
```
Comment (Ghommie) [inline code/datums/station_traits/neutral_traits.dm:592]:
```suggestion
	trait_type = STATION_TRAIT_NEUTRAL
	cost = STATION_TRAIT_COST_LOW
```

IMO, the cost should be low

**Review by Ghommie — COMMENTED** [review]
Also, what sadboysuss said.

**Comment by optimumtact** [comment]
I'm sorry but i'm really not interested in station traits that change the visual look of a department.

**Comment by Singul0** [comment]
rip, it is what it is

**Pull request closed without merging**